jib-core/src/test/java/com/google/cloud/tools/jib/registry/RegistryClientTest.java

```diff
@@ -211,6 +211,29 @@ public void testConfigureBasicAuth()
         registry.getInputRead(), CoreMatchers.containsString("Authorization: Basic dXNlcjpwYXNz"));
   }
 
+  @Test
+  public void testPullManifest()
+      throws IOException, InterruptedException, GeneralSecurityException, URISyntaxException,
+          RegistryException {
+    String imageTag = "testIMage";
```

Comment:
```suggestion
```
I think we can just specify this spring when calling `pullManifest(...)`.

Reply:
Done